deb-get

```diff
@@ -2453,7 +2457,7 @@ case "${ACTION}" in
         ls -lh "${CACHE_DIR}/";;
     clean)
         elevate_privs
-        ${ELEVATE} rm -v "${CACHE_DIR}"/*.deb;;
+        ${ELEVATE} rm -v "${CACHE_DIR}"/*.deb
```

Comment:
Please refrain from changing things outside of the scope of your pull request. It helps keep things organized.

Reply:
Ah, my apologies. I should have made a separate commit for that but I believe there's now another PR with the same correction.